configure

```diff
@@ -69,7 +69,7 @@ fi
 # set defaults before processing command line options
 LDCONFIG=${LDCONFIG-"ldconfig"}
 LDFLAGS=${LDFLAGS-"-L."}
-LDSHAREDLIBC="${LDSHAREDLIBC--lc}"
+LDSHAREDLIBC="${LDSHAREDLIBC}"
```

Comment:
Looks good. If somebody actually needs `-lc` they can just set it before calling configure.

Reply:
Agreed.